sdks/go/alpha_test.go

```diff
@@ -36,26 +36,66 @@ func TestAlphaGetAndSetPlayerCapacity(t *testing.T) {
 	capacity, err := a.GetPlayerCapacity()
```

Comment:
I think there should be `FeaturePlayerTracking` enabled for this test:
```
err := runtime.ParseFeatures(string(runtime.FeaturePlayerTracking) + "=true")
				assert.NoError(t, err)
```
WDYT?

Reply:
I'm going to disagree. This is testing against a mock, of which there is only very light logic, that's also not controlled by a feature flag.

There is no need for feature flags at the SDK level either, since this is all controlled by the backing SDK Server.

So I don't see any reason to have this here? Unless I'm missing something.

Reply:
Thanks for the explanation. Agree with you.